recipe/meta.yaml

```
@@ -21,22 +21,28 @@ requirements:
     - python  >=3.7,<3.11
     - pip
   run:
-    - python  >=3.7,<3.11
+    - python  >=3.7
 
 test:
   imports:
-    - backports
+    - backports.datetime_fromisoformat
   commands:
     - pip check
   requires:
+    # to make sure we're actually testing the backport
+    - python <3.11
```

Could be:
```suggestion
    - python >=3.7,<3.11
```
if we think that's tidier.